en/developer_guide/entity_list_ajax_pagination.rst

```diff
@@ -266,6 +266,49 @@ To specify another folder, you must change the configuration :
                         namespace_prefix: "My\\AppBundle"
                         path: @AppBundle/Ressources/config/mymapping
 
+Custom search field of columns
+------------------------------
+
+By default, Open Orchestra provides differents search field for columns (text, date, number and boolean). You can specify the types of fields in data attribute ``input-header`` in link of navigation panel (futher information in document of `navigation panel`_).
+
+To create a custom search field, you should create an backbone view and add it in the ``tableFieldViewconfigurator``.
+
+For instance with the field text:
+
+.. code-block:: javascript
+
+    TextFieldSearchView = AbstractSearchFieldView.extend(
+
+      events:
+        'keyup input.search-column': 'searchColumn'
+
+      initialize: (options) ->
+        @options = @reduceOption(options, [
+          'column'
+          'domContainer'
+          'table'
+        ])
+        @loadTemplates [
+          'OpenOrchestraBackofficeBundle:BackOffice:Underscore/table/tableHeader/searchField/tableTextField'
+        ]
+        return
+
+      render: ->
+        @setElement @renderTemplate('OpenOrchestraBackofficeBundle:BackOffice:Underscore/table/tableHeader/searchField/tableTextField',
+            column : @options.column
+        )
+        @insertFieldInHeader()
+    )
+
+    ((tableFieldViewconfigurator) ->
+      tableFieldViewconfigurator.text = TextFieldSearchView
+      return
+    ) window.tableFieldViewconfigurator = window.tableFieldViewconfigurator or {}
+
+    You can extend ``AbstractSearchFieldView`` to facilitate research and insertion of the field in the header.
+
+    The key in window.tableFieldViewconfigurator defines the name used in data-attribute (``input-header``).
```

Comment:
You could mention that it is possible to create a completly dynamic field, not only static ones. By sending an ajax request in the view, the server can interact with the rendering of the search field.